models/pipeline.js

```diff
@@ -62,7 +67,7 @@ module.exports = {
      * @type {Joi}
      */
     get: Joi.object(mutate(MODEL, [
-        'id', 'scmUri', 'createTime', 'admins'
+        'id', 'scmUri', 'scmContext', 'createTime', 'admins'
```

Comment:
Okay I'm fine with this. We just need to do the migration right after this is pulled in.